### PR TITLE
fix: v5 dist/reset.css dropped in production mode

### PR DESCRIPTION
**Pull request by murphy-wei**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---
#### 根据文档  
`import { ConfigProvider } from 'antd'`   
`import 'antd/dist/reset.css'`

开发环境正常，生产环境下并没有引入该样式文件. 

webpack说明：
>Note that any imported file is subject to tree shaking. This means if you use something like css-loader in your project and import a CSS file, it needs to be added to the side effect list so it will not be unintentionally dropped in production mode:

<https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free>

### CODE REVIEW

**Comment by yoyo837** [comment]
image

**Comment by yoyo837** [comment]
手误

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "locale"
   ],
   "sideEffects": [
+    "dist/*",
     "*.less"
```
[the remark this answers is not in the document]
Comment (valone-1) [reply package.json:41]: 👌

File: package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "locale"
   ],
   "sideEffects": [
-    "*.less"
+    "dist/*"
```
Comment (yoyo837) [inline package.json:40]: 是否*.css就可以了?

Comment (valone-1) [reply package.json]: done 🫡

**Review by yoyo837 — APPROVED** [review]

**Pull request merged**